app/Site/AppServiceProvider.php

```diff
@@ -55,6 +56,10 @@ public function boot(): void
             $schedule->command(DeleteOverdueUserAccounts::class)->daily();
         });
 
+        Blade::if('hasfeature', function ($feature) {
```

Comment:
Now I'm not sure, does it allow for camel case? like `hasFeature`?

Reply:
It does allow for camel case, but I wasn't sure if that was the way to go because of existing built-ins like `@foreach`. Seems the convention for these directives is to lowercase everything.

Reply:
ah, yes, that makes sense.